solr/docker/templates/Dockerfile.body.template

```diff
@@ -71,7 +71,7 @@ RUN set -ex; \
 
 RUN set -ex; \
     apt-get update; \
-    apt-get -y install acl dirmngr lsof procps wget netcat gosu tini jattach; \
+    apt-get -y install acl lsof procps wget netcat gosu tini jattach; \
```

Comment:
I ran `gradlew testDocker` and all tests passed even without `dirmngr` in the image. So I removed it...